site/config.toml

```diff
@@ -4,7 +4,7 @@ title = "Trunk | Build, bundle & ship your Rust WASM application to the web"
 
 compile_sass = true
 build_search_index = true
-generate_feed = true
```

Comment:
Blast from the past, BTW. I added this 4 years ago to make RSS feed support for the blog. I noticed there is no more blog though, so maybe I do another PR to remove all the blog-specific templating and this config value?